src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + getArea()
+                + " sq. units, radius: "
+                + radius
+                + " units, color: "
+                + getColor());
```

Comment:
Instead of directly printing to the console within the draw method, consider using a return statement to provide the string representation of the object. This would make the code more flexible and easier to test.